lms/djangoapps/courseware/block_render.py

```diff
@@ -670,8 +664,6 @@ def get_block_for_descriptor_internal(user, block, student_data, course_id, trac
         ],
     )
 
-    block.scope_ids = block.scope_ids._replace(user_id=user.id)
```

Comment:
We don't need this - it's already handled in `bind_for_student`: https://github.com/openedx/edx-platform/blob/0b455e0336fa54199fd992c2ce83a81c07dd07c2/xmodule/x_module.py#L630-L631